echopype/echodata/combine.py

```diff
@@ -474,24 +756,17 @@ def combine_echodata(
         groups (e.g. "Sonar/Beam_group1") and values as a list of channel names to select
         within that beam group. The rest of the ``EchoData`` groups with a ``channel`` dimension
         will have their selected channels chosen automatically.
-    consolidated: bool
-        Flag to consolidate zarr metadata.
-        Defaults to ``True``
 
     Returns
     -------
     EchoData
-        A lazy loaded ``EchoData`` object obtained from ``zarr_path``,
+        A lazy loaded ``EchoData`` object,
```

Comment:
Not about this line, but I couldn't highlight it above: could you remove the sentence about `zarr_path` in the first section of the docstring? Thanks!